app/serializers/rest/status_serializer.rb

```diff
@@ -45,7 +45,7 @@ def in_reply_to_account_id
   end
 
   def current_user?
-    !current_user.nil?
+    defined?(current_user) && !current_user.nil?
```

Comment:
great! i will look around for how to do that, just didn't know what to search for since the link is sufficiently abstract to me (not used to ruby)

Reply:
done in https://github.com/mastodon/mastodon/pull/32620/commits/22ed008e4c8f9b92bbe550a80f671a88dc067b95

modified the way options were passed to serializers in helpers to avoid a bunch of special case kwargs, should be equivalent by default